apps/client/src/common/components/view-params-editor/constants.ts

```diff
@@ -263,6 +263,126 @@ export const LOWER_THIRDS_OPTIONS: ParamField[] = [
   },
 ];
 
+export const LOWER_THIRD_OPTIONS: ParamField[] = [
+  {
+    id: 'upper-src',
+    title: 'Upper Text',
+    description: '',
+    type: 'option',
+    values: {
+      title: 'Title',
+      subtitle: 'Subtitle',
+      presenter: 'Presenter',
+      notes: 'Notes',
+      lower: 'Lower Third',
+    },
+  },
+  {
+    id: 'lower-src',
+    title: 'Lower Text',
+    description: '',
+    type: 'option',
+    values: {
+      subtitle: 'Subtitle',
+      title: 'Title',
+      presenter: 'Presenter',
+      notes: 'Notes',
+      lower: 'Lower Third',
+    },
+  },
+  {
+    id: 'upper-colour',
+    title: 'Upper Text Colour',
+    description: 'Upper text colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: '0000ff (default)',
+  },
+  {
+    id: 'lower-colour',
+    title: 'Lower Text Colour',
+    description: 'Lower text colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: '0000ff (default)',
+  },
+  {
+    id: 'upper-bg',
+    title: 'Upper Background Colour',
+    description: 'Upper text background colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: 'dddddd44 (default)',
+  },
+  {
+    id: 'lower-bg',
+    title: 'Lower Background Colour',
+    description: 'Lower text background colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: 'dddddd44 (default)',
+  },
+  {
+    id: 'upper-font',
+    title: 'Upper Text Font',
+    description: 'https://developer.mozilla.org/en-US/docs/Web/CSS/font',
+    type: 'string',
+    placeholder: 'bold 4vh sans-serif',
+  },
+  {
+    id: 'lower-font',
+    title: 'Lower Text Font',
+    description: 'https://developer.mozilla.org/en-US/docs/Web/CSS/font',
```

Comment:
Is this maybe too advanced for a setting? at this level of customisation you might as well use the css override

Reply:
yes.
What should we expose here in this menu?

Reply:
sorry, didnt see this. It is something we still wonder?